src/svelte/ChecklistGroup.svelte

```diff
@@ -12,14 +12,26 @@
   export let lookAndFeel: LookAndFeel
   export let app: App
   export let onToggle: (id: string, type: "page" | "tag") => void
+  export let groupNameAsClass: string
+
+  $: {
+    const groupName = group.groupName || mainTag
+    const sanitzedGroupName = groupName.replace(/[^A-Za-z0-9]/g, "")
+    const dasherizedGroupName = sanitzedGroupName.replace(/^([A-Z])|[\s\._](\w)/g, function (_, p1, p2) {
+      if (p2) return "-" + p2.toLowerCase()
+      return p1.toLowerCase()
+    });
+
+    groupNameAsClass = `group-${dasherizedGroupName}`
+  }
```

Comment:
So this will add a dasherized group name. So #todo/BirthdayWisthList will result in a class name of `group-birthday-wish-list`. This just allows someone to set custom background colors and whatnot, if they wanted to in their custom CSS for specific groups.

Reply:
Cool, this is a nice touch